packages/@coorpacademy-components/src/molecule/wizard-summary/index.js

```diff
@@ -0,0 +1,158 @@
+import React, {useMemo} from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import {NovaSolidComputersSdCard} from '@coorpacademy/nova-icons';
+import Link from '../../atom/link';
+import style from './style.css';
+
+const buildItemsOfSection = items => {
+  return items.map((item, index) => {
+    const {type} = item;
+    switch (type) {
+      case 'mainElement':
+        return (
+          <li key={`mainElement-${index}`}>
+            <span className={style.title}>{item.title}</span>
+            <span className={style.value}>{item.value}</span>
+          </li>
+        );
+      case 'content': {
+        const {category} = item;
+        return (
+          <li className={style.contentWrapper} key={`content-${index}`}>
+            <span className={style.contentCounter}>{index + 1}</span>
+            <div className={style.contentItem}>
+              <div className={style.content}>
+                <span
+                  className={classnames(
+                    category === 'base' && style.base,
+                    category === 'advanced' && style.advanced,
+                    category === 'coach' && style.coach,
+                    category === 'chapter' && style.chapter,
+                    category === 'scorm' && style.scorm,
+                    category === 'video' && style.video,
+                    category === 'article' && style.article,
+                    category === 'podcast' && style.podcast
+                  )}
+                >
+                  {item.label}
+                </span>
+                <span className={style.contentTitle}>{item.title}</span>
```

Comment:
Pour avoir un tooltip (vu qu'il y a un ellipsis)
```suggestion
                <span className={style.contentTitle} title={item.title}>{item.title}</span>
```

Reply:
le titre d'un cours peux être en html (cours en arabe alignement à droite)